index.js

```diff
@@ -22,9 +22,15 @@ getActiveView(panel).setAttribute('backdrag', true);
 
 panel.port.on('link', opts => {
   var title = opts.title;
+  var ytTab = 'https://youtube.com/watch?v=' + parseYoutubeId(opts.src) + '&t=' + opts.time;
+  var vineTab = getVinePageURL(opts.src);
```

Comment:
these should be moved down into the `isYoutube` and `isVine` check since they are only used within those blocks